testing-app/Dockerfile

```
@@ -2,20 +1,18 @@
+COPY package.json ./
+COPY package-lock.json ./
+COPY entrepreneur-ai-assistant-0.1.0.tgz ./
 
-# 5. Copy the rest of the code
-COPY . .
+# Instalar solo dependencias necesarias para producción
+RUN npm install --omit=dev
 
-# 6. Build the application (for React / Next.js)
+# Build de la app Next.js
 RUN npm run build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)